### PR TITLE
wvt_SetRGBColorByString() Color by Number and correct background colo…

### PR DESCRIPTION
**Pull request by JoseQuintas**
wvt_SetRGBColorByString() Color by Number and correct background color when there are more colors on string

### CODE REVIEW

**Comment by JoseQuintas** [comment]
Why "pull not ready" ?

File: contrib/gtwvg/paint.prg [hunk 1]
```diff
@@ -603,6 +603,10 @@ FUNCTION wvt_GetRGBColorByString( cColor, nForeBack )
             s := SubStr( cColor, 1, n - 1 )
          ELSE
             s := SubStr( cColor, n + 1 )
+            s := SubStr( cColor, n + 1 )
+            IF "," $ s
+               s := Substr( s, 1, At( ",", s ) - 1 )
```
Comment (tuffnatty) [inline contrib/gtwvg/paint.prg]: I'd use `Left` here, but if you prefer `SubStr`, then please write it in camel-case.

**Comment by tuffnatty** [comment]
Oops, forgot to finish my review, is it more evident now? Also, I am not sure if color numbers are meant to be supported in color strings, see http://www.oohg.org/cl53/ng9c7f1.html:

>      .  Color numbers: SETCOLOR() supports color letter combinations,
>         but not color number combinations.

**Comment by JoseQuintas** [comment]
Check on SET COLOR

https://harbour.github.io/doc/clc53.html#set-color-cmd

There is a color table including numbers

**Comment by JoseQuintas** [comment]
I was confuse about color 0, but palette users 0-15
when letter, index is n - 1, then color N = 0

**Pull request closed without merging**